zap/src/main/java/org/parosproxy/paros/extension/ExtensionHook.java

```diff
@@ -215,7 +215,7 @@ public void addSessionListener(SessionChangedListener listener) {
      *     removed in a future release.
      */
     @Deprecated
-    public void addSiteMapListner(SiteMapListener listener) {
+    public void addSiteMapListener(SiteMapListener listener) {
```

Comment:
Oops. I need to drop this.

Would you be ok w/ me adding `/*sic*/` between the function name and the arguments? And possibly changing the implementation to call the correctly spelled function?

Reply:
```suggestion
    public void addSiteMapListner/*sic*/(SiteMapListener listener) {
```

Reply:
Is that to appease the tool? Does it have to be exactly like that or can have a space before the comment? (Formatter will add one.)
I'm ok with changing the implementation.

Reply:
It's to talk to humans (i.e. me). I can add a space if that makes people happy.

My tool gives me a list of tokens to check. I then grep for each token. If I see a `sic`, I'm more likely to recognize that the token is intentionally misspelled.

Reply:
> It's to talk to humans (i.e. me).

Isn't the method being deprecated and pointing to the correct method enough?

> I can add a space if that makes people happy.

That's to appease the build/formatter more than make people happy.

Reply:
Sadly my default grep asks for 0 lines of context because I'm looking for the general use of a token in a repository.

Generally, I should review changes in context and notice @deprecated, mia culpa for not. – I was rushing out the door for the day.